src/components/inputs/textField/textField.component.tsx

```diff
@@ -112,74 +117,150 @@ export const TextField: FC<TextFieldProps> = React.forwardRef(
       rows,
       minRows,
       maxRows,
+      labelLeft,
     },
     ref
   ) => {
     return (
       <FormControl className={styles.formControl} fullWidth variant="outlined">
-        {label && (
-          <MuiInputLabel
-            variant="standard"
-            className={styles.label}
-            htmlFor={id}
-            disableAnimation
-            shrink
-            required={required}
+        {label && labelLeft ? (
+          <Box
             sx={{
-              color: 'common.black',
-              '&.Mui-focused': {
-                color: 'common.black',
-              },
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: 'flex-end',
             }}
           >
-            {label}
-          </MuiInputLabel>
-        )}
-        <MuiOutlinedInput
-          id={id}
-          placeholder={placeholder}
-          className={styles.textField}
-          value={value}
-          defaultValue={defaultValue}
-          disabled={disabled}
-          error={error}
-          fullWidth
-          required={required}
-          onChange={handleChange}
-          inputRef={ref}
-          name={name}
-          type={type}
-          multiline={multiline}
-          rows={rows}
-          minRows={minRows}
-          maxRows={maxRows}
-          inputProps={{
-            pattern,
-          }}
-          startAdornment={
-            icon ? (
-              <InputAdornment position="start">{icon}</InputAdornment>
-            ) : null
-          }
-          sx={{
-            '& .MuiOutlinedInput-notchedOutline': {
-              borderColor: ({ palette }) => palette.grey[200],
-            },
-            '&:hover .MuiOutlinedInput-notchedOutline': {
-              borderColor: ({ palette }) => palette.grey[200],
-            },
-            '&.Mui-focused .MuiOutlinedInput-notchedOutline': {
-              borderColor: 'primary.main',
-            },
-            '&.Mui-error .MuiOutlinedInput-notchedOutline': {
-              borderColor: 'error.main',
-            },
-          }}
-        />
-        {helperText && (
-          <FormHelperText error={error} id={`${id}-helper-text`}>
-            {helperText}
-          </FormHelperText>
+            <MuiInputLabel
+              variant="standard"
+              className={`${styles.label} ${styles.labelLeft}`}
+              htmlFor={id}
+              disableAnimation
+              required={required}
+              sx={{
+                color: 'common.black',
+                '&.Mui-focused': {
+                  color: 'common.black',
+                },
+              }}
```

Comment:
I know this has been here before, but can you now replace it with CSS variables? I know that there are others below, but let's at least, fix one at a time. Ignore the rest ;)

Thanks

Reply:
Done :)